apps/legacy-api/__tests__/controllers/PoolPairController.test.ts

```diff
@@ -153,3 +153,64 @@ it('/v1/listyieldfarming', async () => {
     ]
   })
 })
+
+describe('getsubgraphswaps', () => {
+  it.skip('/v1/getsubgraphswaps', async () => {
+    const res = await apiTesting.app.inject({
+      method: 'GET',
+      url: '/v1/getsubgraphswaps'
+    })
+
+    const response = res.json()
+
+    expect(response.data.swaps.length).toStrictEqual(100)
+
+    for (const swap of response.data.swaps) {
+      expect(swap).toStrictEqual({
+        id: expect.stringMatching(/[a-zA-Z0-9]{64}/),
+        pair: {
+          fromToken: {
+            decimals: 8,
+            symbol: expect.any(String),
+            tradeVolume: expect.stringMatching(/^[0-9]+(\.[0-9]{8})?$/)
+          },
+          toToken: {
+            decimals: 8,
+            symbol: expect.any(String),
+            tradeVolume: expect.stringMatching(/^[0-9]+(\.[0-9]{8})?$/)
+          }
+        },
+        timestamp: expect.stringMatching(/\d+/),
+        fromAmount: expect.stringMatching(ONLY_DECIMAL_NUMBER_REGEX),
+        toAmount: 'TODO' // TODO(eli-lim)
+      })
+    }
```

Comment:
shouldn't skip, should test it?

Speaking of this mapping format, should we map to follow this standard as well? On top of what we have

Reply:
Woops - skipped it originally because of local jest timing out. unskipped and changed the response format.